specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceCreateOrUpdateSkillset.json

```diff
@@ -8,212 +8,272 @@
 		"skillset": {
 			"name": "demoskillset",
 			"description": "Extract entities, detect language and extract key-phrases",
-			"skills": [{
-				"@odata.type": "#Microsoft.Skills.Text.NamedEntityRecognitionSkill",
-				"categories": ["organization"],
-				"defaultLanguageCode": "en",
-				"inputs": [{
-					"name": "text",
-					"source": "/document/content"
-				}],
-				"outputs": [{
-					"name": "organizations",
-					"targetName": "organizations"
-				}]
-			},
-			{
-				"@odata.type": "#Microsoft.Skills.Text.LanguageDetectionSkill",
-				"inputs": [{
-					"name": "text",
-					"source": "/document/content"
-				}],
-				"outputs": [{
-					"name": "languageCode",
-					"targetName": "languageCode"
-				}]
-			},
-			{
-				"@odata.type": "#Microsoft.Skills.Text.SplitSkill",
-				"textSplitMode": "pages",
-				"maximumPageLength": 4000,
-				"inputs": [{
-					"name": "text",
-					"source": "/document/content"
-				},
-				{
-					"name": "languageCode",
-					"source": "/document/languageCode"
-				}],
-				"outputs": [{
-					"name": "textItems",
-					"targetName": "pages"
-				}]
-			},
-			{
-				"@odata.type": "#Microsoft.Skills.Text.KeyPhraseExtractionSkill",
-				"context": "/document/pages/*",
-				"inputs": [{
-					"name": "text",
-					"source": "/document/pages/*"
-				},
+			"skills": [
 				{
-					"name": "languageCode",
-					"source": "/document/languageCode"
-				}],
-				"outputs": [{
-					"name": "keyPhrases",
-					"targetName": "keyPhrases"
-				}]
-			}]
-		}
-	},
-	"responses": {
-		"200": {
-			"body": {
-				"name": "demoskillset",
-				"description": "Extract entities, detect language and extract key-phrases",
-				"skills": [{
 					"@odata.type": "#Microsoft.Skills.Text.NamedEntityRecognitionSkill",
```

Comment:
Do you mind finding and replacing in the examples usage of "NamedEntityRecognitionSkill" to "EntityRecognitionSkill"... I must've forgot to do this as part of my swagger update where I added entity recognition skill

Reply:
The files that require this find and replace are
SearchServiceCreateOrUpdateSkillset.json
SearchServiceCreateSkillset.json
SearchserviceGetSkillset.json
SearchServiceListSkillsets.json

Reply:
Sure

Reply:
I attempted this, but it broke Swagger validation, so I rolled it back. It's up to you and your team to update this.